rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -29,19 +29,33 @@
 ASSETS_IN_V2_GLOBALDB = 3095
 
 
-def _count_v2_v3_assets_inserted() -> int:
-    """Counts and returns how many assets are to be inserted by globaldb_v2_v3_assets.sql"""
-    assets_inserted_by_update = 0
+def _count_sql_file_sentences(file_name: str, skip_statements: int = 0):
+    """
+    Count the sql lines in scripts used during upgrades. If the skip_tables argument is provided
```

Comment:
Yabir my man I told you to rename it. That means also:

1. in the docstring
2. Inside the function. `skipped_tables` should be `skipped_statements`